directus-extension-charging-stations-bundle/src/hook-on-create-charging-station/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { defineHook } from '@directus/extensions-sdk';
+
+export default defineHook(({ action }, { env }) => {
+
+    action('ChargingStations.items.create', (input) => {
+        console.log("Subscribing " + input.key + " to connect and close events");
+        const stationId = input.key;
+        const subscriptionUrl = `http://citrine:8080/data/ocpprouter/subscription`;
```

Comment:
Is this something the we should externalize and make configurable somehow?

Reply:
Already the plan! The config.js I mention in the 'WIP' section is for this.